bundles/org.openhab.binding.onkyo/src/main/resources/ESH-INF/thing/tx-nr3007.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onkyo" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="TX-NR3007">
+		<label>Onkyo TX-NR3007 AV Receiver</label>
+		<description>Network enabled Onkyo AV Receivers</description>
```

Comment:
```suggestion
		<description>Network enabled Onkyo AV Receiver</description>
```

Reply:
All the Onkyo other models have ```<description>Network enabled Onkyo AV Receivers</description>``` (with the s).

I was assuming that it was used a group description and hence plural. 

```sh
$ grep '<description' tx*.xml | sort
tx-nr3007.xml:          <description>Network enabled Onkyo AV Receivers</description>
tx-nr414.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr474.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr509.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr515.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr525.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr535.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr545.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr555.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr575e.xml:          <description>Network enabled Onkyo AV Receivers</description>
tx-nr575.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr616.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr626.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr636.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr646.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr656.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr676.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr686.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr708.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr717.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr727.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr737.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr747.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr818.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr828.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr838.xml:           <description>Network enabled Onkyo AV Receivers</description>
```

In any case, shouldn't we make this the same as all the others? I will gladly submit a PR that changes them all to singular.

Reply:
Any thoughts @J-N-K ?

Reply:
Sorry, didn‘t see your comment. They should all be the same. I would prefer the singular for all.